manifests/p/Parsec/Parsec/150.97.3.0/Parsec.Parsec.installer.yaml

```
@@ -0,0 +1,21 @@
+# Automatically updated by the winget bot at 2025/Jan/28
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Parsec.Parsec
+PackageVersion: 150.97.3.0
+InstallerLocale: en-US
+InstallerType: nullsoft
+InstallerSwitches:
+  Silent: /silent /norun /nocleanuser
+  SilentWithProgress: /silent /nocleanuser
+UpgradeBehavior: deny
+ReleaseDate: 2024-03-05
+RequireExplicitUpgrade: true
+AppsAndFeaturesEntries:
+- DisplayVersion: "150-93b"

```

```suggestion
- DisplayVersion: "150-97b"
```
Purely guessing here.